### ISSUE TITLE
训练问题

### ISSUE THREAD

**Issue by Jiejie955** [gangweix/pixel-perfect-depth#1]
亲爱的作者您好，我想复现该代码，请问您使用的是什么设备训练的？

**Comment by FUJI-W** [comment]
@np-csu May I ask if you implemented the training code yourself? If so, would it be possible for you to share it? I am a beginner in this area but find your work very interesting. Thanks in advance.

**Comment by np-csu** [comment]
> [@np-csu](https://github.com/np-csu) May I ask if you implemented the training code yourself? If so, would it be possible for you to share it? I am a beginner in this area but find your work very interesting. Thanks in advance.

@FUJI-W You can build PixelPerfectDepth based on LotusV1 project. Add log depth data loader and loss function from this paper.
